packages/import-utils/src/create-sandbox/templates.ts

```diff
@@ -64,6 +66,10 @@ export function getTemplate(
 
   const nuxt = ["nuxt", "nuxt-edge", "nuxt-ts", "nuxt-ts-edge"];
 
+  if (totalDependencies.indexOf("@adonisjs/framework") > -1) {
```

Comment:
One veeeery small nitpick, could you move it up 2 lines so the `nuxt` variable is next to the `if`?